design/httpproxy-status-meta.md

```diff
@@ -0,0 +1,52 @@
+# HTTPProxy Status Metadata
+
+Status: Draft
+
+## Abstract
+Since the addition of `Includes` for HTTPProxy, it can be difficult for owners of child HTTPProxy resources to understand what has been included to them without having access to a parent HTTPProxy.
+ 
+Users should have a way to understand this extra metadata about the ingress resources. 
+In addition, with this information, additional tooling can be developed against HTTPProxy resources to drive dashboards (i.e. Octant) and other `kubectl plugins`.
+
+## Goals
+- Expose extra metadata information to HTTPProxy resources
```

Comment:
I think it would be helpful to discuss use cases for different kinds of metadata. For example, exposing the vhost is helpful for debugging; I'd also suggest exposing whether authentication is enabled, and whether the vhost is TLS.

Reply:
I have the vhost in the design currently, but could add a section for use-cases. 

Yeah auth is probably a good addon to understand which routes have it enabled and which are disabled, etc.

Reply:
Yes, I'd like some more detail about the use cases you're trying to hit. Why those ones first? What other ones can you think of? Why not those this time? How are you going to allow for growth in the future?